src/main/java/uk/gov/digital/ho/hocs/document/service/DocumentConversionService.java

```diff
@@ -49,7 +54,12 @@ public void convert(MultipartFile file, OutputStream outputStream) {
 
                 DocumentFormat supportedJodFormat = DefaultDocumentFormatRegistry.getFormatByExtension(fileExtension);
 
-                if (imageDocumentConverter.isSupported(fileExtension)) {
+                if (fileExtension.equalsIgnoreCase("pdf")){
```

Comment:
Hardcoded magic string? Might be worth extracting out even if its not likely to change...